smack-core/src/main/java/org/jivesoftware/smack/packet/AbstractError.java

```diff
@@ -85,7 +88,12 @@ public String getDescriptiveText() {
      * @return the descriptive text or null.
      */
     public String getDescriptiveText(String xmllang) {
-        return descriptiveTexts.get(xmllang);
+        try {
```

Comment:
If we say we never use `null` as key, we simply but a  `Objects.requireNotNull(xmllang)` guard before the `get()`. This would also avoid the try/catch block.